bottlecap

```diff
@@ -0,0 +1,80 @@
+#!/bin/sh
+
+# bottlecap, 'cause it's kinda like cork... get it?
+
+usage() {
+	echo "usage:"
+	echo "bottlecap [--dev] [--runtime $runtime] [--build-dir $build_dir]  [--container $container] assemblerargs..."
+	echo "  --dev: rebuild and reinstall mantle, the build scripts, and the ostree-releng-scipts before running coreos-assembler"
+	echo "  --runtime: (auto) specify whether to use podman or docker. Defaults to podman if it is in \$PATH"
+	echo "  --build-dir: Where to put build artifacts, defaults to the current directory"
+	echo "  --container: which coreos-assembler container to use"
+	echo "  --help: print this help message"
+	echo "Command to be executed:"
+	runtime="echo $runtime"
+}
+
+dev=0
+help=0
+runtime="podman"
+which podman &> /dev/null || runtime="docker"
+build_dir="$(pwd)"
+container="quay.io/cgwalters/coreos-assembler"
+
+TEMP=$(getopt -o 'dr:b:c:h' --long 'dev,runtime:,build-dir:,container:,help' -- $@)
```

Comment:
you need to check the exit code of `getopt` and call usage() and exit if it failed.